app/models/rook.rb

```diff
@@ -1,6 +1,19 @@
 class Rook < Piece
   def move_legal?(to_row, to_col)
-    return true if col == to_col || row == to_row
-    false
+    col == to_col || row == to_row
+  end
+
+  def can_castle_to?(king_col)
+    # Jacob and Jim
+    #   I have spent the majority of my time with the task on move_obstructed? for a castle
+    #   Jim had suggested a path_is_clear method
+    #      BUT why do that when PIECE::move_obstructed is perfect
+    #   Jacob recommended overriding super in King to always return true
+    #   This is the only place to call the parent move_obstructed and keep my code DRY
```

Comment:
There is an additional rule: https://en.wikipedia.org/wiki/Castling
See rule 5 in the Requirements section.